wiki/workflows.md

```diff
@@ -0,0 +1,18 @@
+## lint workflow
+Mainly to run the formatters, golangci-lint and staticcheck 
+
+
+## test workflow
+Runs against terraform matrix (only version 1.0.8) and only runs the tests in internal/provider directory
+
+
+TODO: check terraform versions
+
+## terratests workflow
+Runs the integration tests, it requires yggdrasil installation
+
```

Comment:
It runs also the mock tests

Reply:
I double checked, it runs with the `make unittests` target, I updated the docs